addOns/encoder/encoder.gradle.kts

```diff
@@ -6,10 +6,20 @@ description = "Adds encode/decode/hash dialog and support for scripted processor
 zapAddOn {
     addOnName.set("Encoder")
     addOnStatus.set(AddOnStatus.BETA)
-    zapVersion.set("2.9.0")
```

Comment:
This can be kept, the add-on does not need to be compiled with the snapshot (does not use any new features).